python/ray/_private/usage/usage_lib.py

```diff
@@ -237,6 +238,37 @@ def _put_library_usage(library_usage: str):
             logger.debug(f"Failed to write a library usage to the home folder, {e}")
 
 
+def record_extra_usage_tag(key: str, value: str):
```

Comment:
Should we just accept a dictionary to reduce # of blocking calls for this API?

Reply:
Currently kv doesn't have a batch put api anyway. If there is perf issue, we can add later.